st2actions/st2actions/container/base.py

```diff
@@ -96,18 +96,15 @@ def _do_run(self, runner, runnertype_db, action_db, actionexec_db):
             runner.pre_run()
 
             LOG.debug('Performing run for runner: %s', runner)
-            if isinstance(runner, AsyncActionRunner):
-                (status, result, query_context) = runner.run(action_params)
-            else:
-                (status, result) = runner.run(action_params)
+            (status, result, context) = runner.run(action_params)
```

Comment:
Feels like we combined context and query_context. The confusing part for me is how would a runner know if something is simply context or query_context - is there any difference?